vendor/imports.go

```diff
@@ -50,11 +50,16 @@ func ParseImports(root string) (map[string]bool, error) {
 
 // FetchMetadata fetchs the remote metadata for path.
 func FetchMetadata(path string, insecure bool) (io.ReadCloser, error) {
-	schemes := []string{"https", "http"}
+	schemes := []string{"https"}
```

Comment:
@hyper-carrot  I'm sorry this isn't right. Now the user will always get a message saying "skipping insecure propocol", even if it is never required that the server hit the insecure protocol.

I'll try to fix this